BlueShift-iOS-SDK/BlueShiftAlertView.m

```diff
@@ -13,8 +13,12 @@ - (UIAlertController *)alertViewWithPushDetailsDictionary:(NSDictionary *)pushDe
     
     NSDictionary *pushAlertDictionary = [pushDetailsDictionary objectForKey:@"aps"];
     NSString *pushCategory = [[pushDetailsDictionary objectForKey:@"aps"] objectForKey:@"category"];
-    NSString *pushMessage = [pushAlertDictionary objectForKey:@"alert"];
-    UIAlertController *blueShiftAlertController = [UIAlertController alertControllerWithTitle:kAlertTitle message:pushMessage preferredStyle:UIAlertControllerStyleAlert];
+    NSString *pushTitle = [[pushAlertDictionary objectForKey:@"alert"] objectForKey:@"title"];
+    if (!pushTitle || [pushTitle isEqualToString:@""]) {
+        pushTitle = kAlertTitle;
+    }
+    NSString *pushMessage = [[pushAlertDictionary objectForKey:@"alert"] objectForKey:@"body"];
+    UIAlertController *blueShiftAlertController = [UIAlertController alertControllerWithTitle:pushTitle message:pushMessage  preferredStyle:UIAlertControllerStyleAlert];
```

Comment:
do we need to do nil check on `pushMessage`?

Reply:
There isn't any need to check for the pushMessage, if its nill then pushMessage will be set nil. The alertController method accepts nil input for message.
The pushMessage is compulsory field on the dashboard, while the pushTitle is optional.